config/environments/development.rb

```diff
@@ -70,3 +70,10 @@
   # Apply autocorrection by RuboCop to files generated by `bin/rails generate`.
   # config.generators.apply_rubocop_autocorrect_after_generate!
 end
+
+# to warn re: n+1 queries or unused eager loading
+config.after_initialize do
+  Bullet.enable = true
+  Bullet.alert = true
+  Bullet.rails_logger = true
+end
```

Comment:
Do you think this would be helpful to put in `test.rb` too? I haven't used this gem, so maybe it's not recommended for some reason, but seems like it would be nice to get some notice when you're tests run?